hugr-core/src/import.rs

```diff
@@ -938,7 +956,7 @@ impl<'a> Context<'a> {
             model::Term::FuncType { .. } => Err(error_unsupported!("`(fn ...)` as `TypeParam`")),
 
             model::Term::ListType { item_type } => {
-                let param = Box::new(self.import_type_param(*item_type)?);
+                let param = Box::new(self.import_type_param(*item_type, TypeBound::Any)?);
```

Comment:
Not sure why this is any (also not covered by tests), is this a todo waiting for a list constraint?

Reply:
The item type is nested within the list and since we only constrain parameters directly here, we can not express that the items of the list should be copyable. Therefore the `TypeBound::Any`.